app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -391,7 +391,9 @@ $(document).on('change', '.virtual', function() {
 function construct_host_name() {
   var host_name_el = $('#host_name')
   var host_name = host_name_el.val();
-  if (host_name_el.data('appendDomainNameForHosts') === false) {
+  if (host_name_el.data('appendDomainNameForHosts') === false ||
```

Comment:
If you remove `append_domain_name_for_hosts` from host's form, this entry is no longer available.

Reply:
What do you mean by not _longer available_? Is the field disabled?